composer.json

```diff
@@ -20,6 +20,7 @@
   ],
   "require": {
     "silverstripe/framework": "^4@dev",
+    "intervention/image": "^2.3",
```

Comment:
We should get rid of `suggest: ext-gd` below, as that's intervention's responsibility.